code/game/objects/items/granters/magic/blind.dm

```diff
@@ -17,3 +17,27 @@
 	. = ..()
 	to_chat(user, span_warning("You go blind!"))
 	user.blind_eyes(10)
+
+/obj/item/book/granter/action/spell/blind/wgw
+	name = "Woody's Got Wood"
+	pages_to_mastery = 69 // Andy's favorite number
+	uses = 0 // it's spent
+	desc = "This book looks dangerious. Only suffering awaits those who read."
+	remarks = list( // Death awaits
+		"One day while Andy was masturbating, Woody got wood.",
```

Comment:
I appreciate your enthusiasm but this needs to load from a config file and NOT be present on the repo

because otherwise we will trip up the automated filters.

Reply:
From the headcoder:

> there is no chance in hell wgw is going to be in an in game log bro

Even if I put it as a separate file, it's not going to get used. 

The way I setup the book wasn't meant to be read directly anyway.  It spawns used, but if by chance someone is able to use the charge spell on it, you still have to read 69 pages before you can learn blind.  (ten minutes of reading time with no interruptions)

Reply:
well then remove it?